build.gradle

```diff
@@ -3,27 +3,24 @@
 buildscript {
     ext {
         kotlin_version = '1.3.72'
-        jacocoVersion = '0.1.4'
-        junitVersion = '4.12'
-        jupiter_version = '5.5.1'
-        mannodermausVersion = '1.5.1.0'
+        mannodermaus_version = '1.5.1.0'
```

Comment:
Removed unused extra properties. Also changes from `mannodermausVersion` to `mannodermaus_version ` - always a good idea to have the same pattern (I could see something_version is being most used here, so the way to go now).

Reply:
Yes, it is fine for now. We are migrating it to a Kotlin file into the `buildSrc` project.